forge16/src/main/java/com/envyful/vaults/player/PlayerVault.java

```diff
@@ -36,7 +35,7 @@ public class PlayerVault {
     private ItemStack display;
 
     public PlayerVault(int id) {
-        this(id, "vault#" + (id + 1));
+        this(id, "Vault #" + (id + 1));
```

Comment:
If we're modifying the default then it may as well be in the config

Reply:
Sure, will add